viz-lib/src/visualizations/visualizationsSettings.js

```diff
@@ -32,8 +32,13 @@ HelpTrigger.defaultValues = {
   children: null,
 };
 
+function Link(props) {
+  return <a {...props} />;
+}
+
 export const visualizationsSettings = {
   HelpTriggerComponent: HelpTrigger,
+  LinkComponent: Link,
```

Comment:
Missing replace this component in https://github.com/getredash/redash/blob/68b70ed63b2e38e15b6e7a875077bdf810b1d553/client/app/components/visualizations/visualizationComponents.jsx#L12-L34

Reply:
https://github.com/getredash/redash/pull/5122/commits/072b2be2c9dc5f7db56594fd1616bffc84a86576